pkg/services/config.go

```diff
@@ -542,6 +544,7 @@ func New(ctx context.Context, config Config) (*Services, error) {
 		RetentionPolicy:            retentionPolicy,
 		MCPCatalog:                 config.MCPCatalogs,
 		AllowedMCPDockerImageRepos: config.AllowedMCPDockerImageRepos,
+		MCPCatalogRefreshKick:      make(chan struct{}),
```

Comment:
nit: can we make this a function instead that does the right thing? That way callers can just call the function instead of
```go
select {
case services.MCPCatalogRefreshKick <- struct{}{}:
default:
}
```

Reply:
I implemented this, but I wasn't sure how to do it without having both a channel and a function as fields on the `Services` object. Let me know what you think. The change is pushed.